sdks/java/extensions/sorter/pom.xml

```diff
@@ -97,6 +97,10 @@
                   <pattern>org.apache.hadoop</pattern>
                   <shadedPattern>org.apache.beam.repackaged.org.apache.hadoop</shadedPattern>
                 </relocation>
+                <relocation>
+                  <pattern>com.google.common</pattern>
+                  <shadedPattern>org.apache.beam.repackaged.com.google.common</shadedPattern>
+                </relocation>
```

Comment:
You have to do a bit more to shade Guava, because the jar actually includes stuff not in the `com.google.common` path. See the core-java module for details.